config/globals.ts

```diff
@@ -1,9 +1,10 @@
 const globals = (env?: ENV): Globals => {
   const appEnv = process.env.APP_ENV || env || 'development';
+  const nodeEnv = (env == null || env === 'development') ? 'development' : 'production';
```

Comment:
waarom 2x ==?

Reply:
`== null` is een shortcut voor `=== null || === undefined` http://adripofjavascript.com/blog/drips/equals-equals-null-in-javascript.html